### PR TITLE
feat: #43 setup husky

### PR DESCRIPTION
**Pull request by Ekaterina1994**

### CODE REVIEW

File: .husky/commit-msg [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+if ! head -1 "$1" | grep -qE "^(feat|fix|docs|test|style|refactor): #[0-9]+ .*"; then
+    echo "❌ Ooops! Aborting commit. Your commit message is invalid. Follow the pattern, please" >&2
```
Comment (tritonJS826) [inline .husky/commit-msg:5]: 1. add comment about commits to convention
2. make error transparent

File: mw-webapp/src/component/table/columns.tsx [hunk 2]
```diff
@@ -55,7 +55,7 @@ const getDateValue = (cellValue: CellContext<DayReport, Date>) => {
   );
 };
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any
+// eslint-disable-next-line @typescript-eslint/no-explicit-any, max-len
```
Comment (tritonJS826) [inline mw-webapp/src/component/table/columns.tsx:58]: remove

File: mw-webapp/src/model/businessModel/DayReport.ts [hunk 3]
```diff
@@ -45,8 +44,9 @@ export class DayReport {
 
   /**
    * Mentor's comments
```
Comment (tritonJS826) [inline mw-webapp/src/model/businessModel/DayReport.ts:46]:
```suggestion
   * Mentor's comments uuids
```

**Review by tritonJS826 — APPROVED** [review]

File: mw-webapp/src/component/table/columns.tsx (hunk 2 above)
Comment (Ekaterina1994) [reply mw-webapp/src/component/table/columns.tsx:58]: Done

File: mw-webapp/src/model/businessModel/DayReport.ts (hunk 3 above)
Comment (Ekaterina1994) [reply mw-webapp/src/model/businessModel/DayReport.ts:46]: Done

**Pull request merged**